src/pages/ConnectWallet/components/WalletList.tsx

```diff
@@ -0,0 +1,168 @@
+import { Alert, AlertDescription, AlertIcon, Center, Spinner, Stack } from '@chakra-ui/react'
+import { useQuery } from '@tanstack/react-query'
+import { useCallback, useMemo, useState } from 'react'
+import { useHistory } from 'react-router'
+import { MobileWalletDialogRoutes } from 'components/MobileWalletDialog/types'
+import { WalletActions } from 'context/WalletProvider/actions'
+import { KeyManager } from 'context/WalletProvider/KeyManager'
+import { useLocalWallet } from 'context/WalletProvider/local-wallet'
+import { MobileConfig } from 'context/WalletProvider/MobileWallet/config'
+import { getWallet, listWallets } from 'context/WalletProvider/MobileWallet/mobileMessageHandlers'
+import type { RevocableWallet } from 'context/WalletProvider/MobileWallet/RevocableWallet'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+import { WalletCard } from './WalletCard'
+
+type MobileWalletDialogProps = {
+  footerComponent?: JSX.Element
+  isEditing?: boolean
+}
+
+export const MobileWallestList: React.FC<MobileWalletDialogProps> = ({
+  footerComponent,
+  isEditing,
+}) => {
+  const { dispatch, getAdapter, state } = useWallet()
+  const { walletInfo } = state
+  const localWallet = useLocalWallet()
+  const history = useHistory()
+  const [error, setError] = useState<string | null>(null)
+
+  const { isLoading, data: wallets } = useQuery({
+    queryKey: ['listWallets'],
+    staleTime: 0,
+    gcTime: 0,
+    refetchOnMount: true,
+    queryFn: async () => {
+      try {
+        const vaults = await listWallets()
+        if (!vaults.length) {
+          setError('walletProvider.shapeShift.load.error.noWallet')
```

Comment:
This translation string doesn't exist:

![image](https://github.com/user-attachments/assets/62365164-7e8d-4b3f-b889-734b9b2e472a)

We should also disable the "Edit" button in this state.

Reply:
This string does exists but we were not translating them...

Removed the edit button if error happened in  [2d9f44b](https://github.com/shapeshift/web/pull/8689/commits/2d9f44b23a0ce70c448c2fbcfaf53654c99ef88f) and [3d4afb4](https://github.com/shapeshift/web/pull/8689/commits/3d4afb44eed27599a5e9015ec6cf45a6ca4ce0a3)